src/platform_impl/linux/x11/window.rs

```diff
@@ -1134,13 +1143,13 @@ impl UnownedWindow {
         let cursor = unsafe {
             // We don't care about this color, since it only fills bytes
             // in the pixmap which are not 0 in the mask.
-            let dummy_color: ffi::XColor = mem::uninitialized();
+            let mut dummy_color = MaybeUninit::uninit();
             let cursor = (self.xconn.xlib.XCreatePixmapCursor)(
                 self.xconn.display,
                 pixmap,
                 pixmap,
-                &dummy_color as *const _ as *mut _,
-                &dummy_color as *const _ as *mut _,
+                dummy_color.as_mut_ptr(),
+                dummy_color.as_mut_ptr(),
```

Comment:
Afaik, `assume_init` should be called on `dummy_color` so that `Drop` is called on it, else we got a memory leak.

Reply:
Even though `dummy_color` is an [`XColor`](https://docs.rs/x11-dl/2.18.3/x11_dl/xlib/struct.XColor.html) instance, containing only plain old data?